apix-integrationtests/src/main/java/eu/xenit/apix/rest/v1/tests/BulkTest.java

```diff
@@ -52,6 +52,10 @@
     @Autowired
     ServiceRegistry serviceRegistry;
 
+    private String jsonObjectGetStringFromInt(JSONObject targetObject, String key) {
+        return String.valueOf(targetObject.getInt(key));
```

Comment:
You could change this to targetObject.optInt(...) so it returns null when the value is not an int

Reply:
Thanks, but if it's not a blocking comment, I rather keep it, since this has to work over different versions of the library and it took quite some builds to get it working for all.